### ISSUE TITLE
Umbrella: OpenGHG Storage Sprint

### ISSUE THREAD

**Issue by gareth-j** [openghg/openghg#867]
# OpenGHG Storage Sprint - 8th and 9th of Jan

## TODO

1. Add data to the new object store

- flux
- footprints 
- surface

2. Profile

3. Reading

- Retrieving data from the object store
- Creating `ModelScenario` objects
- Using the `get_obs_*` functions
- ?

4. Check storage used

### Profilers

We should profile the storage of the different types
- memray - https://github.com/bloomberg/memray
- scalene - https://github.com/plasma-umass/scalene

## Current data

- Users switch to main before we do the full proper release
- Collate all the data people are using so we can add it to the new object store

**Comment by rt17603** [comment]
May want to change this for the initial version is version 1 "v1" rather than version 0 "v0".

**Comment by rt17603** [comment]
At the moment we seem to be unable to add footprint co2 data. Running the following on BP1:

```
from openghg.standardise import standardise_footprint
from pathlib import Path

fp_path = Path("/group/chemistry/acrg/LPDM/fp_NAME/EUROPE/")
fp_filepath = fp_path / "WAO-20magl_UKV_co2_EUROPE_202101.nc"

site = "WAO"
species = "co2"
domain = "EUROPE"
model = "NAME"
metmodel = "UKV"
inlet = "20magl"

standardise_footprint(fp_filepath, site=site, species=species, domain=domain, model=model, metmodel=metmodel, inlet=inlet)
```

Produces the error:
```
ValueError: Codec does not support buffers of > 2147483647 bytes
```
Full error log attached: [co2_footprint_codec_error.log](https://github.com/openghg/openghg/files/13954018/co2_footprint_codec_error.log)

This seems to mean that we are currently unable to write files above a certain size (buffer at least) to a zarr object store.

**Comment by rt17603** [comment]
> At the moment we seem to be unable to add footprint co2 data. 
> ...
> For reference, profiling using memray, this reaches 17GiB and the specific "WAO-20magl_UKV_co2_EUROPE_202101.nc" file is 144MB in storage size. (Doesn't seem to allow upload of html so unable to currently add the flame graph for this - can supply upon request).

This should be fixed by Issue #908

**Comment by rt17603** [comment]
Covered by other issues
